drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/radio_test/fmac_api.h

```diff
@@ -245,11 +246,12 @@ void nrf_wifi_fmac_dev_rem_rt(struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx);
  */
 enum nrf_wifi_status nrf_wifi_fmac_dev_init_rt(struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx,
 #if defined(CONFIG_NRF_WIFI_LOW_POWER) || defined(__DOXYGEN__)
-			int sleep_type,
+					       int sleep_type,
```

Comment:
Please move unrelated changes to a separate commit.

Reply:
This change is for adding a function parameter...the realignment is done along with that.

Reply:
This change is for adding a new function parameter...the realignment is done along with the addition.